pkg/client/unversioned/request.go

```diff
@@ -380,13 +382,13 @@ func (r *Request) FieldsSelectorParam(s fields.Selector) *Request {
 		return r
 	}
 	s2, err := s.Transform(func(field, value string) (newField, newValue string, err error) {
-		return fieldMappings.filterField(r.groupVersion, r.resource, field, value)
+		return fieldMappings.filterField(*r.content.GroupVersion, r.resource, field, value)
```

Comment:
Minor nit: could you just change this method to take a pointer so the calls look better?